eth_tester/main.py

```diff
@@ -244,15 +273,25 @@ def get_nonce(self, account, block_number="latest"):
     #
     # Blocks, Transactions, Receipts
     #
+
+    def _get_pending_transaction_by_hash(self, transaction_hash):
+        for transaction in self._pending_transactions:
```

Comment:
Not very performant. If that is a concern, can make `_pending_transaction` a dict with key=hash value=transaction

Reply:
Eh, it'd need to be an ordered dict to preserve transaction order right?  I think that would be fine, but it's not a big deal.  The performance hit here isn't likely to be measurable against the actual evm execution overhead.

Reply:
Not a concern, at least for now. If someone wants to test a really high pending count, we can revisit.